Models/updateVotes.models.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection.js");
+
+function updateVotes(inc_votes, article_id) {
+  const updateVotesQuery = `UPDATE articles 
+  SET votes = votes + $1 
+  WHERE article_id = $2
+  RETURNING *;`;
+
+  return db
+    .query(updateVotesQuery, [inc_votes, article_id])
+
+    .then((result) => {
+      return result.rows[0];
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+}
```

Comment:
There should be no catch block here as it will stop the one in your controller from being executed. Please delete this one 🙂